lib/diagnostic.js

```diff
@@ -19,15 +19,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = false; // replace null with your response
 //
 
 // Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = ['alderaan', 'deathStar', 'global'];
```

Comment:
`['deathStar', 'fire']`
Same as above, `fire` is defined within the deathStar scope, so it has access to the variables in that scope.
Because `laserQuantity` is declared inside a function, it's not available in the global scope.